exla/Makefile

```
@@ -44,8 +44,8 @@ $(EXLA_SO): $(EXLA_CACHE_SO)
 		cp -a $(abspath $(XLA_EXTENSION_LIB)) $(EXLA_LIB_DIR) ; \
 		cp -a $(abspath $(EXLA_CACHE_SO)) $(EXLA_SO) ; \
 	else \
-		ln -sf $(abspath $(XLA_EXTENSION_LIB)) $(EXLA_LIB_DIR) ; \
-		ln -sf $(abspath $(EXLA_CACHE_SO)) $(EXLA_SO) ; \
+		ln -sf $(shell realpath --relative-to $(PRIV_DIR) $(XLA_EXTENSION_LIB)) $(EXLA_LIB_DIR) ; \
```

I wonder if just passing the relative path for `XLA_EXTENSION_LIB` directly would work, since realpath isn't available:

```suggestion
		ln -sf $(PRIV_DIR)/../$(XLA_EXTENSION_LIB) $(EXLA_LIB_DIR) ; \
```

---

I guess it'd depend if PRIV_DIR is a relative path in itself though

---

Yeah, I wasn't sure if PRIV_DIR was an abs path